.github/workflows/compiler-zoo.yml

```diff
@@ -79,6 +82,10 @@ jobs:
             cc: clang-17,
             distro: ubuntu-22.04,
             llvm-ppa-name: jammy
+          }, {
+            cc: clang-18,
+            distro: ubuntu-22.04,
+            llvm-ppa-name: jammy
```

Comment:
why dont you simply use `distro: ubuntu-24.04` here?

Reply:
did you happen to run any of the ubsan ci tests with this update?  We have several reports of ubsan hitting lots of failures with clang-18.  See https://github.com/openssl/openssl/issues/22896

Reply:
Ah, interesting.
FWIW I have the clang-18 already enabled for a while in my openssl-111-feature branch:
https://github.com/bernd-edlinger/openssl/blob/ed1e44493c6faec6c7e71fd5cc0dff2eeaec8a4d/.github/workflows/compiler-zoo.yml#L80-L81
And indeed I have got reports of that kind, but only for one day or so.
While I was already testing a commit that added `-fno-sanitize=function` to the ubsan build,
I noticed that the github actions somehow stopped reporting that, probably because those
warnings did cause more confusion than they are worth at all, so they restored the previous behvior.